src/sparseml/yolov8/trainers.py

```diff
@@ -571,7 +604,20 @@ def export(self, **kwargs):
             for p in self.model.parameters():
                 p.requires_grad = True
             manager = ScheduledModifierManager.from_yaml(one_shot)
-            manager.apply(self.model)
+
+            # TODO: I hate the fact that to scale this functionality to other tasks
+            # we might have to (worst case scenario) copy and paste many
+            # arguments from the train.py. This is not ideal.
```

Comment:
Can we not just arguments for data and device to export CLI?

Reply:
I guess that's what you mean by copy and paste? Yes, we would essentially have to maintain two pathways: one-shot and training-aware with both of them having similar support.